frontend/src/metabase/admin/databases/components/DatabaseEditApp/Sidebar/Sidebar.unit.spec.tsx

```diff
@@ -68,22 +65,7 @@ function setup({
   };
 }
 
-function mockMetabaseSettings() {
-  const original = MetabaseSettings.get.bind(MetabaseSettings);
-  const spy = jest.spyOn(MetabaseSettings, "get");
-  spy.mockImplementation(key => {
-    if (key === "site-uuid") {
-      return Utils.uuid();
-    }
-    return original(key);
-  });
-}
-
 describe("DatabaseEditApp/Sidebar", () => {
-  beforeAll(() => {
-    mockMetabaseSettings();
```

Comment:
Is the reason why can remove this mock because this test doesn't rely on `MetabaseSettings.get("site-uuid")` anymore?